scripts/tests/test_validate_docstrings.py

```diff
@@ -1004,6 +1006,32 @@ def test_item_subsection(self, idx, subsection):
         assert result[idx][3] == subsection
 
 
+class TestDocstringClass(object):
+    @pytest.mark.parametrize('name_and_expected_obj',
+                             [('pandas.isnull', pandas.isnull),
+                              ('pandas.DataFrame', pandas.DataFrame),
+                              ('pandas.Series.sum', pandas.Series.sum)])
+    def test_resolves_class_name(self, name_and_expected_obj):
+        name, expected_obj = name_and_expected_obj
+        d = validate_docstrings.Docstring(name)
+        assert d.obj is expected_obj
+
+    @pytest.mark.parametrize('invalid_name', ['panda', 'panda.DataFrame'])
+    def test_raises_for_invalid_module_name(self, invalid_name):
+        # When an invalid module name is supplied, an ImportError should be
+        # raised, and the message should contain the user-supplied name.
+        msg = 'No module can be imported.*{}'.format(invalid_name)
+        with pytest.raises(ImportError, match=msg):
+            validate_docstrings.Docstring(invalid_name)
+
+    @pytest.mark.parametrize('invalid_name',
+                             ['pandas.BadClassName',
+                              'pandas.Series.bad_method_name'])
+    def test_raises_for_invalid_attribute_name(self, invalid_name):
+        with pytest.raises(AttributeError):
```

Comment:
Can you add the expected message here?

Reply:
If you can address this should be good to merge

Reply:
Hmm alright I suppose might be overkill here. Can always address in follow up PRs for the issue you linked